__tests__/articles.test.js

```diff
@@ -0,0 +1,53 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const app = require("../app");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed({ topicData, userData, articleData, commentData });
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/articles/:article_id", () => {
+  test("200 - GET - will return an article object with the given article_id, with the properties of author, title, article_id, body, topic, created_at, votes and article_img_url", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        const article = res.body.article;
+        expect(article).toHaveProperty("article_id", expect.any(Number));
+        expect(article).toHaveProperty("title", expect.any(String));
+        expect(article).toHaveProperty("topic", expect.any(String));
+        expect(article).toHaveProperty("author", expect.any(String));
+        expect(article).toHaveProperty("body", expect.any(String));
+        expect(article).toHaveProperty("created_at", expect.any(String));
+        expect(article).toHaveProperty("votes", expect.any(Number));
+        expect(article).toHaveProperty("article_img_url", expect.any(String));
+      });
```

Comment:
Fundamentally this test needs to be re written. You are specifically requesting article 1, however your server could send article 12 and this would still pass. You are absolutely not expecting any response for this, you are expecting article 1 and the relevant data.

Reply:
<img width="1408" alt="Screenshot 2024-08-26 at 13 14 04" src="https://github.com/user-attachments/assets/2b21b776-5ac3-45cd-8341-5d6156d3f455">
<img width="982" alt="Screenshot 2024-08-26 at 13 14 11" src="https://github.com/user-attachments/assets/4656d18f-6244-4d26-bbcc-38d5d9eb75ae">